port/unix/omrsysinfo.c

```diff
@@ -253,6 +253,18 @@ static intptr_t omrsysinfo_get_aix_ppc_description(struct OMRPortLibrary *portLi
 #define __power_10() (_system_configuration.implementation == POWER_10)
 #endif /* !defined(__power_10) */
 
+#if !defined(__power_11)
+#define POWER_11 0x80000 /* Power 11 class CPU */
+#define __power_11() (_system_configuration.implementation == POWER_11)
+#endif /* !defined(__power_11) */
+
+/*
+ * Please update macro below to keep in sync with the latest POWER processor as OMR knew
+ * about. such that, CPU recognition is not that fragile as in the past. As this macro
+ * stands now, any later processors are recognized as POWER11 at least.
```

Comment:
sentence structure and grammar needs to be fixed here:
```
 * Please update the macro below to stay in sync with the latest POWER processor known to OMR,
 * ensuring CPU recognition is more robust than in the past. As the macro currently stands, any
 * later processors are recognized as at least POWER11.
```

Reply:
i can change to that